<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurseforgeModBrowser</class>
 <widget class="QMainWindow" name="CurseforgeModBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>259</y>
      <width>93</width>
      <height>19</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Mod Loader:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="loaderSelect">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>102</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchText">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>160</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Search mod on Curseforge...</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="sortSelect">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>167</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Popularity</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Last Updated</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Name</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Author</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Total Downloads</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Category</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Game Version</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="MenuToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="searchBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Mod">
    <property name="title">
     <string>&amp;Mod</string>
    </property>
    <widget class="QMenu" name="menuDownload">
     <property name="title">
      <string>Download</string>
     </property>
     <property name="icon">
      <iconset theme="download">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
    <addaction name="actionOpen_Curseforge_Mod_Dialog"/>
    <addaction name="menuDownload"/>
    <addaction name="actionOpen_Website_Link"/>
    <addaction name="actionCopy_Website_Link"/>
   </widget>
   <widget class="QMenu" name="menu_Path">
    <property name="title">
     <string>&amp;Path</string>
    </property>
    <widget class="QMenu" name="menuSelect_Game_Version">
     <property name="title">
      <string>Select Game Version</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuSelect_Category">
     <property name="title">
      <string>Select Category</string>
     </property>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuSelect_Game_Version"/>
    <addaction name="menuSelect_Category"/>
    <addaction name="separator"/>
    <addaction name="actionMod"/>
    <addaction name="actionModpacks"/>
    <addaction name="actionTexturepacks"/>
    <addaction name="actionWorld"/>
   </widget>
   <addaction name="menu_Mod"/>
   <addaction name="menu_Path"/>
  </widget>
  <widget class="QToolBar" name="categoryToolBar">
   <property name="windowTitle">
    <string>Categories</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="versionToolBar">
   <property name="windowTitle">
    <string>Game Versions</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionMod">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mod</string>
   </property>
  </action>
  <action name="actionWorld">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>World</string>
   </property>
  </action>
  <action name="actionModpacks">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Modpacks</string>
   </property>
  </action>
  <action name="actionTexturepacks">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Texturepacks</string>
   </property>
  </action>
  <action name="actionCopy_Website_Link">
   <property name="icon">
    <iconset theme="edit-copy">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Copy Website Link</string>
   </property>
  </action>
  <action name="actionOpen_Curseforge_Mod_Dialog">
   <property name="icon">
    <iconset resource="../../images/image.qrc">
     <normaloff>:/image/curseforge.svg</normaloff>:/image/curseforge.svg</iconset>
   </property>
   <property name="text">
    <string>Open Curseforge Mod Dialog</string>
   </property>
  </action>
  <action name="actionOpen_Website_Link">
   <property name="text">
    <string>Open Website Link</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MenuToolBar</class>
   <extends>QToolBar</extends>
   <header location="global">ui/menutoolbar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../images/image.qrc"/>
 </resources>
 <connections/>
</ui>
